### PR TITLE
feat: add syllablic forms and counts for lines

### PR DESCRIPTION
**Pull request by bhajneet**
### Summary of PR
Added syllable forms for each word and the line's syllable count at the end. Also cleaned up padding of line header and refactored gurmukhi font to be used anywhere as a class.

Before:
<image src="https://user-images.githubusercontent.com/14130567/98255947-fba78480-1f4b-11eb-8532-3a9ce46f0b21.png" width="600">

After: 
<image src="https://user-images.githubusercontent.com/14130567/98255895-ed596880-1f4b-11eb-98b8-4a1744e35f66.png" width="600">

### Tests for unexpected behavior
- Various sources/pages

### Time spent on PR
2 hours

### Reviewers
@Harjot1Singh

### CODE REVIEW

**Comment by bhajneet** [comment]
Fixing this rq for mobile:

image

**Comment by bhajneet** [comment]
Done

image

File: frontend/src/components/LineView.css [hunk 1]
```diff
@@ -10,21 +10,31 @@
...
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.line-view .header h1 div {
+  display: flex;
+  flex-direction: column;
+  margin-right: .35em;
```
Comment (Harjot1Singh) [inline frontend/src/components/LineView.css:35]: Please append 0

File: frontend/src/index.css [hunk 2]
```diff
@@ -15,3 +15,6 @@ code {
     monospace;
 }
 
+.gurmukhi {
+  font-family: 'Open Gurbani Akhar Bold';
+}
```
Comment (Harjot1Singh) [inline frontend/src/index.css:20]: Missing EOF line break

File: frontend/src/components/LineView.js [hunk 3]
```diff
@@ -122,23 +122,27 @@ const LineView = ( {
...
+                      rel="noreferrer"
+                    >
+                      {word}
+                    </a>
+                    <span>{toSyllabicSymbols( word )}</span>
+                  </div>
                 ) )
                 .reduce( ( prev, curr ) => [ prev, ' ', curr ] )
               : <Loader size="1em" />}
+            {gurmukhi && ( <span className="syllable-count">{countSyllables( gurmukhi )}</span> )}
```
Comment (Harjot1Singh) [inline frontend/src/components/LineView.js:145]: How have you lined up the syllables, since they don't appear to be grouped to a word?

**Review by Harjot1Singh — CHANGES_REQUESTED** [review]

File: frontend/src/components/LineView.js (hunk 3 above)
Comment (bhajneet) [reply frontend/src/components/LineView.js:145]: The final count is not grouped to the word, but yes it's lined up as the image shows in the rounded square icon

File: frontend/src/index.css (hunk 2 above)
Comment (bhajneet) [inline frontend/src/index.css:20]:
```suggestion
}

```

File: frontend/src/components/LineView.css (hunk 1 above)
Comment (bhajneet) [inline frontend/src/components/LineView.css:35]:
```suggestion
  margin-right: 0.35em;
```

File: frontend/src/components/LineView.js (hunk 3 above)
Comment (Harjot1Singh) [reply frontend/src/components/LineView.js:145]: I see, I thought it was somehow the individual word counts, my bad. Please add a newline above it just to visually separate it from the "words" code block above

Comment (bhajneet) [reply frontend/src/components/LineView.js:145]:
```suggestion

            {gurmukhi && ( <span className="syllable-count">{countSyllables( gurmukhi )}</span> )}
```

**Comment by bhajneet** [comment]
Partially failing because this doesn't exist https://viewer.shabados.com/sources/8/page/2/line/0

**Comment by bhajneet** [comment]
I don't know how to fix these tests

**Comment by saihaj** [comment]
I will fix that test. It was partially successful when the tests PR got merged. That is possible because I cherry picked #309 into #317 and something got messed up

**Comment by Harjot1Singh** [comment]
Hot Potato

**Comment by bhajneet** [comment]
> 
> 
> I will fix that test. It was partially successful when the tests PR got merged. That is possible because I cherry picked #309 into #317 and something got messed up

More so that the page button should not exist there, since there is no page 2. It should be disabled (currently a user can get to a broken page otherwise)

**Comment by saihaj** [comment]
image
When I run this locally `npm run test` it is working fine. I looked at the video output from the artifacts of test job and the error it shows is not related to tests. Some state is `null` for some reason 
image

**Comment by saihaj** [comment]
There is probably a race condition or just something wrong during execution. After re running the job it is passing ✅ . If we get this error again then only worth exploring more on this.

**Review by Harjot1Singh — APPROVED** [review]
The race condition for the non-existent page 2 is probably because the app works by attempting to load a page to/assume it exists, and when it doesn't, disable the button.

**Pull request merged**